### ISSUE TITLE
implement moving vote in triangle with ipad gestures

### ISSUE THREAD

**Issue by giovanisp** [fairdemocracy/vilfredo-client#151]
As stated in: https://developer.apple.com/library/ios/documentation/AppleApplications/Reference/SafariWebContent/HandlingEvents/HandlingEvents.html#//apple_ref/doc/uid/TP40006511-SW7  (one finger)

**Comment by pietrosperoni** [comment]
Yes, but I would move it to later. First a system that work, then we make it universally accessible. A system universally accessible but that does not work is less useful that a system that work but is not universally accessible.

**Comment by giovanisp** [comment]
Ok, but if it does not work now, we can't make official releases until we get it working... 
In our next meeting let's review the document we did in our first meeting.
Anyway, ok, you can move it to the timeframe you feel it is important to you respect to the other development priorities.

**Comment by pietrosperoni** [comment]
Let's make a real effort to write down the whole list of priorities and even a realistic (HAHAHA) schedule of what should come out when.

Either this or the question, "how should vilfredo calendar look like". In fact we should first make a meeting to decide what should be developed. And then a Vilfredo question to decide the order, and when. But maybe we solve all in the meeting.

**Comment by giovanisp** [comment]
> Il giorno 29/gen/2015, alle ore 01:28, Pietro Speroni <notifications@github.com> ha scritto:
> Let's make a real effort to write down the whole list of priorities and even a realistic (HAHAHA) schedule of what should come out when.

This is called "roadmap" and is exactly what the milestones and issues on github are :-)
We just need to move then around and give dates. Seems easy, but we need to make sure we can deliver that much work (unfortunately github does not provide time management like JIRA or REDMINE).

> Either this or the question, "how should vilfredo calendar look like". In fact we should first make a meeting to decide what should be developed. And then a Vilfredo question to decide the order, and when. But maybe we solve all in the meeting.
> 
"How should vilfredo roadmap look like".
Yes for the meeting, on feb when you are back in Rome?

Are you willing to "promote" rc1 around now? Or let's keep a low profile and keep working until we feel confident enough to tell people on facebook groups and other mailing lists we are delivering live?

Gs

**Comment by pietrosperoni** [comment]
Is there an external resource for roadmap that we can use and would integrate well?

I am back on the 20 of February. Or I could come next week for 1 day (i.e. carpe diem).

Should I promote Vilfredo? I don't know. Isn't there the risk of burning it? It is already quite crispy ;-)

In any case in theory this sunday morning I should present it at he famous Berlin Community
